packages/qwik/src/core/use/use-store.public.ts

```diff
@@ -19,6 +19,13 @@ export interface UseStoreOptions {
    * Default is `true`.
    */
   reactive?: boolean;
+
+  /**
+   * If `true` then bind method is called on the `newStore` object to ensure
+   * that the `this` context is bound to the store object.
+   * Default is `true`.
+   */
+  bind?: boolean;
```

Comment:
I don't think this is the right approach:

1. Store should not be aware of binding
2. Even if Store does the bind, it will not survive serialization, since serialization does not understand binding.